CMakeLists.txt

```diff
@@ -39,7 +39,7 @@ set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${PROJECT_SOURCE_DIR}/cmake)
 
 # options shared by all targets
 option(VERBOSE_CMAKELISTS "Show extra information while processing CMakeLists.txt files." OFF)
-option(WARNINGS_AS_ERRORS "Treat all compiler warnings as error." OFF)
+option(WARNINGS_AS_ERRORS "Treat any compiler warning as an error (adds -Werror flag)." OFF)
```

Comment:
As in "enabling this option adds -Werror flag."  But of course it could also be "enabling this option will add -Werror flag."  So many options...   :)